src/main/java/de/doubleslash/keeptime/view/SettingsController.java

```diff
@@ -190,16 +191,6 @@ private void initialize() {
 
                alert.showAndWait();
             }
-            if (!displayProjectsRightCheckBox.isSelected() && hideProjectsOnMouseExitCheckBox.isSelected()) {

```

Comment:
can you add logic here, to set the value of hideProjects to false, if it is true. Maybe some users of a previous version have it active. We need to deactivate it then on settings save :)

Reply:
Option is disabled and on save it get set to false 👍